honeybee_radiance_folder/folder.py

```diff
@@ -607,7 +607,7 @@ def combined_receivers(self, folder='receiver', auto_mtx_path=False):
 
         # find the light_path for each grid
         for grid in grids:
-            if not 'light_path' in grid:
+            if not grid['light_path']:
```

Comment:
Good catch. We have to check for 'light_path' first and then check if it is empty.
```suggestion
            if not 'light_path' in grid or not grid['light_path']:
```

Reply:
@mostaphaRoudsari, I just made this change.